storage/reads/aggregate_resultset.go

```diff
@@ -32,11 +32,16 @@ func NewWindowAggregateResultSet(ctx context.Context, req *datatypes.ReadWindowA
 		return nil, errors.Errorf(errors.InternalError, "attempt to create a windowAggregateResultSet with %v aggregate functions", nAggs)
 	}
 
+	ascending := true
+	if len(req.Aggregate) > 0 && req.Aggregate[0].Type == datatypes.AggregateTypeLast {
+		ascending = false
```

Comment:
It would as the logic for this is located in the `tsdb` package. Storage will create two types of TSM cursors, ascending or descending, depending on the value that is set here. Launcher and pipeline tests however do get us test coverage over this part.

Reply:
Ok, that makes sense.